### DISCUSSION TITLE
No real-world results when running hello_world.py file

### DISCUSSION THREAD

**Discussion by sabzer** [USC-ACTLab/crazyswarm#709] [Q&A]
Hello,

We are trying to setup Crazyswarm, but are not successful when trying to run the hello_world.py file. Although we are able to see the pose of the Crazyflie via RViz after running hover_swarm.launch, nothing happens when running hello_world.py, and no errors are returned in terminal. We are using OptiTrack, and vrpn. Print statements in the hello_world.py file show that the "swarm = Crazyswarm()" command does not reach completion. We have investigated NatNet usage and we are able to communicate with the computer supporting motive and receive pose information. We've probably looked at every single related discussion post. Further system information: ROS Noetic, Ubuntu 20.04, Crazyflie 2.1, Motive 2.3. We have been stuck on this error for quite a while, and any help is appreciated. Please let us know if there is any other information we can provide that would help with an answer!!

**Comment by knmcguire** [comment]
Hi! 

Do you need to use Noetic or is ROS2 also possible? There is a Crazyswarm ROS2 port happening now: https://github.com/IMRCLab/crazyswarm2.

It is still in progress and not everything works yet but the support will be quicker there
